src/visitpy/visitpy/PyExportDBAttributes.C

```diff
@@ -5,6 +5,7 @@
 #include <PyExportDBAttributes.h>
 #include <ObserverToCallback.h>
 #include <stdio.h>
+#include "PyDBOptionsAttributes_Helpers.h"
```

Comment:
Just curious but why must this code go into separate files as opposed to just going into `PyDBOptionsAttributes.code`

Reply:
I couldn't control the order of the function defs in a way that would work in the built-in override.